transport/src/test/java/io/camunda/zeebe/transport/stream/impl/messages/SerializationTest.java

```diff
@@ -63,6 +64,20 @@ void shouldSerializeAddStreamRequestWithMetadataWriter() {
     assertThat(deserialized.metadata()).isEqualTo(BufferUtil.wrapString("bar"));
   }
 
+  @Test
+  void shouldSerializeAddStreamResponse() {
+    // given
+    final var response = new AddStreamResponse();
+
+    // when
+    response.write(buffer, 0);
+    final var deserialized = new AddStreamResponse();
+
+    // then
+    assertThatCode(() -> deserialized.wrap(buffer, 0, response.getLength()))
+        .doesNotThrowAnyException();
```

Comment:
💭 These messages/responses have no equal methods I guess?

Reply:
They would if they were not empty :smile: